### PR TITLE
Support for array annotation values

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

**Comment by gsmet** [comment]
@stuartwdouglas if you can have a look at this one and that one https://github.com/quarkusio/gizmo/pull/25 , I can cut a new Gizmo release tomorrow.

File: src/main/java/io/quarkus/gizmo/AnnotationUtils.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package io.quarkus.gizmo;
+
+import org.jboss.jandex.AnnotationValue;
+import org.objectweb.asm.AnnotationVisitor;
+
+public final class AnnotationUtils {
```
Comment (stuartwdouglas) [inline src/main/java/io/quarkus/gizmo/AnnotationUtils.java]: This should probably be package private.

Comment (gytis) [reply src/main/java/io/quarkus/gizmo/AnnotationUtils.java]: @stuartwdouglas done. I made both class and method package-private. Is that OK or should only be a class?

Comment (stuartwdouglas) [reply src/main/java/io/quarkus/gizmo/AnnotationUtils.java]: Perfect.

**Pull request merged**